pipelines/optimus/Optimus.wdl

```diff
@@ -222,6 +222,7 @@ workflow Optimus {
   call Merge.MergeSortBamFiles as MergeSorted {
     input:
       bam_inputs = PreMergeSort.bam_output,
+      sample_id = sample_id,
```

Comment:
I think it might be easier to write if you change `MergeSortBamFiles` to accept a `String` parameter `output_bam_filename`. That way the task can use `output_bam_filename` throughout.

Reply:
That makes sense, I changed it to take output_bam_filename from Optimus wdl